logstash-core/lib/logstash/runner.rb

```diff
@@ -437,12 +438,7 @@ def check_log_configuration_contains_javascript_usage
 
      logger.info("Log4j configuration path used is: #{config_file.path}")
      log_config = File.open(config_file.absolute_path).read
-     if log_config =~ /^[^#]+script\.language\s*=\s*JavaScript/
-       deprecation_logger.deprecated "Logging configuration uses deprecated Script log appender or filter, which will be removed in a future major release of Logstash."
-       return true
-     else
-       return false
-     end
+     (log_config =~ /^[^#]+script\.language\s*=\s*JavaScript/) != nil
```

Comment:
nit: might be easier to read:
```suggestion
     log_config.match? /^[^#]+script\.language\s*=\s*JavaScript/
```